packages/runtime-core/src/apiAsyncComponent.ts

```diff
@@ -154,13 +156,13 @@ export function defineAsyncComponent<
       const delayed = ref(!!delay)
 
       if (delay) {
-        setTimeout(() => {
+        delayTimer = window.setTimeout(() => {
```

Comment:
Do not use `window` directly in runtime-core, because the core is platform-independent, otherwise it will not be able to run in nodejs, which is fatal to ssr.

Reply:
I have fixed this.